pkg/cmd/run/view/view.go

```diff
@@ -153,7 +153,7 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 	cmd.Flags().BoolVar(&opts.Log, "log", false, "View full log for either a run or specific job")
 	cmd.Flags().BoolVar(&opts.LogFailed, "log-failed", false, "View the log for any failed steps in a run or specific job")
 	cmd.Flags().BoolVarP(&opts.Web, "web", "w", false, "Open run in the browser")
-	cmdutil.AddJSONFlags(cmd, &opts.Exporter, shared.RunFields)
+	cmdutil.AddJSONFlags(cmd, &opts.Exporter, shared.SingleRunFields)
```

Comment:
SingleRunFields includes the `jobs` field but RunFields (used by `gh run list`) doesn't